docs/sphinx/applications/python/vqe_advanced.ipynb

```diff
@@ -341,7 +341,7 @@
     "\n",
     "cudaq.set_target(\"nvidia-mqpu\")\n",
     "\n",
-    "num_qpus = 4\n",
+    "num_qpus = 1\n",
```

Comment:
Perhaps this should be:
```suggestion
    "num_qpus = cudaq.get_target().num_qpus()\n",
```
I believe that would allow it to dynamically calculate the number of QPUs to be equal to the number of GPUs on the host system.